string[] joint_names
float64[] positions
float64[] durations
